tasks/main.yml

```diff
@@ -8,11 +8,14 @@
     group: root
     mode: 0644
   when: apt_manage_sources_list
+  notify: update
   tags:
     - configuration
     - apt
     - apt-configuration
 
+- meta: flush_handlers
```

Comment:
yes, you are right.

i tried to add `apt_sources_list_file.changed` to your update task but nothing happens since the `apt_update_cache_valid_time` is in place.

any idea how i can reuse that task?